util/fsdiffer/beforeafter.go

```diff
@@ -0,0 +1,63 @@
+package fsdiffer
+
+import (
+	"fmt"
+	"path/filepath"
+)
+
+// The BeforeAfterFSDiffer is used to generate changes in a given directory
```

Comment:
I made the change in https://github.com/appc/acbuild/pull/104. Once that gets merged I'll rebase this on master and rename the calls in here.